palworld/palworld.json

```diff
@@ -0,0 +1,496 @@
+      "type": "integer",
+      "desc": "If no players from a guild are online for this amount of time (in hours) the guild will be disbanded",
+      "display": "Empty Guild Deletion Time",
+      "required": true,
+      "value": "72"
+    },
+    "BanListURL": {
+      "type": "string",
+      "desc": "Palworld devs provide a list of globally blacklisted players, you can optionally substitute your own list.",
+      "display": "Banned Players List",
+      "value": "https://api.palworldgame.com/api/banlist.txt"
+    },
+    "BaseCampMaxNum": {
+      "type": "integer",
+      "desc": "Max number of total bases allowed in the world",
+      "display": "Max allowed bases in world",
+      "required": true,
+      "value": "128"
+    },
+    "BaseCampWorkerMaxNum": {
+      "type": "integer",
+      "desc": "Maximum Pals that can be assigned to each base. You still need to upgrade your base to unlock each slot.",
+      "display": "Max Worker Pals per base",
+      "required": true,
+      "value": "15"
+    },
+    "BuildObjectDamageRate": {
+      "type": "integer",
+      "desc": "Multiplier to affect incoming damage to buildings ",
+      "display": "Building Damage Multiplier",
+      "required": true,
+      "value": "1.000000"
+    },
+    "BuildObjectDeteriorationDamageRate": {
+      "type": "integer",
+      "desc": "Multiplier to affect how much damage built objects will take",
+      "display": "Building Deterioration Multiplier",
+      "required": true,
+      "value": "1.000000"
+    },
+    "CollectionDropRate": {
+      "type": "integer",
+      "desc": "Multiplier that affects how much items are dropped from nodes such as trees, rocks, and ores",
+      "display": "Node Farming Drop Rate",
+      "required": true,
+      "value": "1.000000"
+    },
+    "CollectionObjectHpRate": {
+      "type": "integer",
+      "desc": "Multiplier that affects how much health each node has before disappearing",
+      "display": "Node Health",
+      "required": true,
+      "value": "1.000000"
+    },
+    "CollectionObjectRespawnSpeedRate": {
+      "type": "integer",
+      "desc": "Multiplier that affects how quickly nodes respawn",
+      "display": "Node Respawn Time",
+      "required": true,
+      "value": "1.000000"
+    },
+    "CoopPlayerMaxNum": {
+      "type": "integer",
+      "desc": "The amount of players allowed to be in a co-op party",
+      "display": "Max Party Size",
+      "required": true,
+      "value": "4"
+    },
+    "DayTimeSpeedRate": {
+      "type": "integer",
+      "desc": "How long each day lasts",
+      "display": "Day Duration",
+      "required": true,
+      "value": "1.000000"
+    },
+    "DeathPenalty": {
+      "type": "string",
+      "desc": "What you drop on death. Valid options are None (keep inventory), Item (drop inventory, keep equipment), ItemAndEquipment (drop inventory and equipment), All (drop inventory, equipment, and pals. this is default for servers)",
```

Comment:
Options are permitted which help create dropdowns for this. https://github.com/pufferpanel/templates/blob/v2.6/minecraft-forge-legacy/minecraft-forge-legacy.json#L55-L84 is an example